compass/landice/tests/antarctic/mesh.py

```diff
@@ -0,0 +1,219 @@
+import numpy as np
+import netCDF4
+import xarray
+from matplotlib import pyplot as plt
+from shutil import copyfile
+
+from mpas_tools.mesh.creation import build_planar_mesh
+from mpas_tools.mesh.conversion import convert, cull
+from mpas_tools.planar_hex import make_planar_hex_mesh
+from mpas_tools.io import write_netcdf
+from mpas_tools.logging import check_call
+
+from compass.step import Step
+from compass.model import make_graph_file
+from compass.landice.mesh import gridded_flood_fill, \
+    set_rectangular_geom_points_and_edges, \
+    set_cell_width, get_dist_to_edge_and_GL
+
+
+class Mesh(Step):
+    """
+    A step for creating a mesh and initial condition for Antarctic test cases
+
+    Attributes
+    ----------
+    mesh_type : str
+        The resolution or mesh type of the test case
+    """
+
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        mesh_type : str
+            The resolution or mesh type of the test case
+        """
+        super().__init__(test_case=test_case, name='mesh')
+
+        self.add_output_file(filename='graph.info')
+        self.add_output_file(filename='Antarctic_8to80km.nc')
+        self.add_input_file(
+            filename='antarctica_8km_2020_10_20.nc',
+            target='antarctica_8km_2020_10_20.nc',
+            database='')
+
+    # no setup() method is needed
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        logger = self.logger
+        config = self.config
+        section = config['antarctic']
+
+        logger.info('calling build_cell_width')
+        cell_width, x1, y1, geom_points, geom_edges, floodFillMask = \
+            self.build_cell_width()
+        logger.info('calling build_planar_mesh')
+        build_planar_mesh(cell_width, x1, y1, geom_points,
+                          geom_edges, logger=logger)
+        dsMesh = xarray.open_dataset('base_mesh.nc')
+        logger.info('culling mesh')
+        dsMesh = cull(dsMesh, logger=logger)
+        logger.info('converting to MPAS mesh')
+        dsMesh = convert(dsMesh, logger=logger)
+        logger.info('writing grid_converted.nc')
+        write_netcdf(dsMesh, 'grid_converted.nc')
+        # If no number of levels specified in config file, use 10
```

Comment:
```suggestion
        # If no number of levels specified in config file, use default
```
(default currently isn't 10)

Reply:
Technically, the default is whatever is specified in the config file, so this comment should probably be removed.